src/includes/enriching-events/set-user/dotnet.nlog.mdx

```diff
@@ -0,0 +1,50 @@
+<Alert level="" title="NLog Configuration">
```

Comment:
I noticed we're currently getting: 
![image](https://user-images.githubusercontent.com/1633368/132774399-767083b3-76ba-4249-b354-428e0d2e2f9c.png)

It's because we're missing snippets here: https://github.com/getsentry/sentry-docs/tree/master/src/includes/enriching-events/unset-user

For .NET it would be just setting it to `null`. Would be nice as a follow up PR 🙏